pyroSAR/drivers.py

```diff
@@ -1549,6 +1549,9 @@ class Archive(object):
         required for postgres driver: host where the database is hosted. Default: 'localhost'
     port: int
         required for postgres driver: port number to the database. Default: 5432
+    add_table_schemas: :obj:`sqlalchemy.schema` or :obj:`list` of  :obj:`sqlalchemy.schema`
```

Comment:
could this be `` :class:`sqlalchemy.schema.Table` ``?

Reply:
yes, my mistake! then maybe we should change also the variable names and docstrings to be consistent with SQLAlchemy?

Reply:
Ah cool. Yes, this would be really great.

Reply:
i changed some things there in the next commit, also testing for added tables.
maybe there should be an add_tables function outside of the init?

Reply:
should be resolved now